src/script/lua_api/l_mainmenu.cpp

```diff
@@ -391,6 +391,37 @@ int ModApiMainMenu::l_get_favorites(lua_State *L)
 			lua_settable(L, top_lvl2);
 		}
 
+		if (server["clients_list"].isArray()) {
+			unsigned int index_lvl2 = 1;
+			lua_pushstring(L,"clients_list");
```

Comment:
Space after comma. The old code wasn't checked for code style.
https://dev.minetest.net/Code_style_guidelines